homeassistant/components/enigma2/media_player.py

```diff
@@ -249,7 +254,7 @@ def device_state_attributes(self):
         currservice_begin:  is in the format '21:00'.
         currservice_end:    is in the format '21:00'.
         """
-        if self.e2_box.in_standby:
+        if self.e2_box.in_standby or self.e2_box.is_offline:
```

Comment:
This isn't needed, as the device is marked unavailable.

```suggestion
        if self.e2_box.in_standby:
```